### ISSUE TITLE
Presets and Profiles on Windows Homedir not used

### ISSUE THREAD

**Issue by cyberpunkbln** [OpenShot/openshot-qt#1476]
**System Details:**
* Operating System / Distro: Windows 10 64 Bit
* OpenShot Version: 2.4.1 64 Bit
* _Please attach log files if crash_

Hello,

OpenShot don't load presets and profiles from C:\Users\CurrentUser\.openshot_qt, but there a presets and profile folders???

thx

**Comment by peanutbutterandcrackers** [comment]
@DylanC - Maybe this, too, has to do with permissions? Like all the other issues that had to do with permissions on Windows?

**Comment by DylanC** [comment]
@peanutbutterandcrackers - Yeah I would think so

**Comment by cyberpunkbln** [comment]
Hi,

hmm OpenShot write and read very well some other Files in the Home-Folder\.openshot_qt like configs, some folder (empty presets and profile folders,too) and read can every Program in Windows from every Location. 

When i put presets and profiles in the Program Files Directory Folder OpenShot reads this, but when i put this presets and profiles under c:\users\CurrentUser\.openshot-qt\presets ... ... then OpenShot don't read the files.

Simple read permissions must OpenShot have and under the home-directory OpenShot have all permissions the the normal User have, it is his home-directory:).

Some other Windows Issues?

**Comment by peanutbutterandcrackers** [comment]
Hmm... that is strange. I would expect other files to be affected, too, if this was really permission related.

Yes, it seemed that for some users, they had to run openshot as administrator to get it to work because for some reason it couldn't work. i.e. permission issues.

@DylanC - Perhaps this requires some testing?

**Comment by DylanC** [comment]
@peanutbutterandcrackers - I will have a look. Maybe its just that Openshot doesn't look in these locations for the files. Maybe not related to the permissions.

**Comment by cyberpunkbln** [comment]
Hello,
@DylanC thanx for your look. It can be an strange permission bug, when a normal user starts with read-permissions an program in programfiles but at the time i think the %localappdata%-permissions are bug-resistent. It will be great when can exclude an internal don't look at this locations;).

**Comment by DylanC** [comment]
@cyberpunkbln @peanutbutterandcrackers - I tested this with the latest daily build and only could see a profiles folder (no presets). I copied an existing profile and gave it a silly name. I could see it then appears in the dropdown. Take a look at this screenshot:

![image](https://user-images.githubusercontent.com/4488048/39150176-61e6cc12-4739-11e8-882f-77520f56b72d.png)

My only advice can be to try the latest daily build or wait until v2.4.2 is released to try again.
